packages/mew_vi/mew_vi.0.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "zandoye@gmail.com"
+authors: [ "ZAN DoYe" ]
+homepage: "https://github.com/kandu/mew_vi"
+bug-reports: "https://github.com/kandu/mew_vi/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/kandu/mew_vi.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "mew"
```

Comment:
Would it be ok if all the packages from this repository were to sync all their versions using the `{= version}` constraint?
It allows packages to be more easily maintainable.

Reply:
Thanks, I added a version constraint.  :)